tests/e2e/vsphere.go

```diff
@@ -56,6 +56,46 @@ const (
 	virtualDiskUUID = "virtualDiskUUID"
 )
 
+func (vs *vSphere) createCnsVolume(ctx context.Context, createSpecList []cnstypes.CnsVolumeCreateSpec) (*object.Task, error) {
```

Comment:
code here in the function is available in govomi? Let's not duplicate code here in this repository.